web/src/main/java/se/inera/intyg/intygsadmin/web/service/status/SendStatusForCareGiverServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.intygsadmin.web.service.status;
+
+import org.springframework.stereotype.Service;
+import se.inera.intyg.intygsadmin.web.integration.WCIntegrationRestService;
+import se.inera.intyg.intygsadmin.web.integration.dto.SendStatusForCareGiverIntegrationRequestDTO;
+
+@Service
+public class SendStatusForCareGiverServiceImpl implements SendStatusForCareGiverService {
+
+    private final WCIntegrationRestService wcIntegrationRestService;
+
+    public SendStatusForCareGiverServiceImpl(WCIntegrationRestService wcIntegrationRestService) {
+        this.wcIntegrationRestService = wcIntegrationRestService;
+    }
+
+    @Override
+    public Integer send(String careGiverId, SendStatusForCareGiverIntegrationRequestDTO request) {
+        final var integrationRequest = SendStatusForCareGiverIntegrationRequestDTO.create(careGiverId, request.getStart(), request.getEnd(),
```

Comment:
This is correct but here you can see an example of where we usually have one request for the controller layer and one for the integration layer. You're creating a SendStatusForCareGiverIntegrationRequestDTO from your request, but right now it's the same type of object